types/webpack-manifest-plugin/webpack-manifest-plugin-tests.ts

```diff
@@ -15,7 +15,7 @@ const options: Options = {
         return file;
     },
     publicPath: "prod",
-    removeKeyHash: /[a-Z0-9]/g,
+    removeKeyHash: /[A-Za-z0-9]/g,
```

Comment:
How this change related to node v22?

Reply:
I'm not sure why the tests are failing, so I tried to fix it